exotica_core/src/kinematic_tree.cpp

```diff
@@ -1014,6 +1052,35 @@ void KinematicTree::SetFloatingBaseLimitsPosXYZEulerZYX(
     UpdateJointLimits();
 }
 
+void KinematicTree::SetFloatingBaseLimitsPosXYZEulerZYX(
+    const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity = {}, const std::vector<double>& acceleration = {})
```

Comment:
Remove the default values (they should, if at all, be placed in the header) - but since we are keeping the old API - please remove, and also from the logic within this function.